### PR TITLE
Refactor projects page [SCI-5134]

### PR DESCRIPTION
**Pull request by urbanrotnik**
Jira ticket: [SCI-5134](https://biosistemika.atlassian.net/browse/SCI-5134)

### CODE REVIEW

File: app/views/projects/index/modals/_new.html.erb [hunk 1]
```diff
@@ -0,0 +1,30 @@
+
```
Comment (okriuchykhin) [inline app/views/projects/index/modals/_new.html.erb:1]: Empty lines?

File: app/controllers/projects_controller.rb [hunk 2]
```diff
@@ -258,18 +225,20 @@ def experiment_archive
     current_team_switch(@project.team)
   end
 
+  def dt_state_load
+    render json: { state: current_team.current_view_state(current_user).state.dig('projects', 'table') }
```
Comment (okriuchykhin) [inline app/controllers/projects_controller.rb:229]: What if there is no current_team?

Comment (urbanrotnik) [reply app/controllers/projects_controller.rb:229]: You are not able to access that screen if you do not have current_team...

Comment (urbanrotnik) [reply app/controllers/projects_controller.rb:229]: Orr.. hm, should I return 403 or something?

File: app/views/projects/index.html.erb [hunk 3]
```diff
@@ -1,169 +1,49 @@
-
 <% provide(:head_title, t("projects.index.head_title")) %>
 
 <% if current_team %>
-  <%= render partial: "shared/sidebar", locals: { page: 'project' } %>
+  <%= render partial: "shared/sidebar", locals: {page: 'project'} %>
```
Comment (okriuchykhin) [inline app/views/projects/index.html.erb:4]: Why did you remove spaces?

Comment (urbanrotnik) [reply app/views/projects/index.html.erb:4]: Auto formater.. ups

File: app/controllers/projects_controller.rb (hunk 2 above)
Comment (okriuchykhin) [reply app/controllers/projects_controller.rb:229]: Maybe just safe navigation?

**Review by okriuchykhin — APPROVED** [review]
